0.Preprocess_Data/1.preprocessing_morphology_data.ipynb

```diff
@@ -898,7 +898,7 @@
                     "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        "µ": "u",
</pre><p>May consider using standard keyboard characters, other than mu. Didn't know this was possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>yep, that is why I am replacing the mu with u. The mu is Unicode I believe </p>